roles/azure_virtual_machine_with_public_ip/meta/argument_specs.yml

```diff
@@ -93,8 +107,6 @@ argument_specs:
             description:
               - Initial disk size in GB for blank data disks. Cannot be larger than 1023 GB.
               - Can only be modified when VM is deallocated.
-          managed_disk_type:
```

Comment:
data_disks.managed_disk_type is valid argument

Reply:
`lun`, `caching`, `disk_size_gb`, `managed_disk_type`, `storage_account_name`, `storage_blob_name`, `storage_container_name` are related to `data_disks`. Should  they be `data_disk` options?

Reply:
Fixed